src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -100,11 +100,19 @@ trait SubmissionComponent {
       val query = for {
         submissionRec <- findByWorkspaceId(workspaceContext.workspaceId)
         userRec <- rawlsUserQuery if (submissionRec.submitterId === userRec.userSubjectId)
-      } yield (submissionRec, userRec)
+        methodConfigRec <- methodConfigurationQuery if (submissionRec.methodConfigurationId === methodConfigRec.id)
+        entityRec <- entityQuery if (submissionRec.submissionEntityId === entityRec.id)
+      } yield (submissionRec, userRec, methodConfigRec, entityRec)
       
-      query.result.flatMap(recs => DBIO.sequence(recs.map { case (submissionRec, userRec) =>
-        loadSubmission(submissionRec.id).map(sub => (sub.get, rawlsUserQuery.unmarshalRawlsUser(userRec)))
-      }))
+      query.result.flatMap{recs => DBIO.sequence(recs.map {
```

Comment:
I haven't been able to get this to work. Inside the for-yield, those vars represent the slick Table classes ... it's only after the query.result mapping that they represent the individual rows within those tables. But, I was able to get rid of the DBIO.sequence and DBIO.successful by using map, not flatMap ... see the third commit in this PR.